lib/rules/no-tracked-properties-from-args.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const ERROR_MESSAGE =
+  'Do not use this.args to create @tracked properties as this will not be updated if the args change. Instead use a getter.';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: 'suggestion',
```

Comment:
If this issue will cause a bug then we can use `problem` instead of `suggestion`.